scripts/test_hiz_oc_bound.py

```
@@ -0,0 +1,226 @@
+    else:
+        metric_cols = [f"acc_{c}" for c in ACC_FIELDS]
+    fieldnames = ["bound", "scene_name"] + metric_cols
+
+    by_bound: dict[str, dict[str, dict[str, str]]] = {}
+    scene_order: list[str] = []
+
+    for bound, path in sources:
+        with path.open(newline="", encoding="utf-8") as fin:
+            reader = csv.DictReader(fin)
+            if reader.fieldnames is None:
+                raise ValueError(f"No header in {path}")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)